piet-svg/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "piet-svg"
+version = "0.1.0"
```

Comment:
I struggle with what to do with version numbers for these crates. Eventually we'll all go to 0.1.0 together, but maybe just for the sake of pragmatism we can start this at `0.0.7`, just to make my search & replace job easier? If that offends your sensibilities then I'm happy with `0.0.1`, but I want to save `0.1.0`.

Reply:
Done.